app/models/admin/action_log_filter.rb

```diff
@@ -76,7 +76,7 @@ def scope_for(key, value)
     when 'account_id'
       Admin::ActionLog.where(account_id: value)
     when 'target_account_id'
-      account = Account.find(value)
+      account = Account.find_or_initialize_by(id: value)
```

Comment:
I would perhaps just change to `find_by(id: ...)` and use `&.` below. Don't like the idea of initializing an object just for one access!

Reply:
If the account previously existed but then got deleted, `Account.find_by(id: value)` would return `nil`, and thus wouldn't find the logs that do exist for that deleted account. Initializing the object was in order to concisely fix that (since the relationship to `target` is polymorphic).

I don't think that case can currently occur when using only Mastodon-provided tools, so maybe that's not worth it.